hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/CacheInfoCollector.java

```diff
@@ -0,0 +1,38 @@
+package com.hazelcast.internal.util.phonehome;
+
+import com.hazelcast.cache.impl.CacheService;
+import com.hazelcast.config.CacheSimpleConfig;
+import com.hazelcast.core.DistributedObject;
+import com.hazelcast.instance.impl.Node;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.stream.Collectors.toList;
+
+public class CacheInfoCollector implements MetricsCollector {
+
+    Collection<DistributedObject> caches;
+
+    @Override
+    public Map<String, String> computeMetrics(Node hazelcastNode) {
+
+        Collection<DistributedObject> distributedObjects = hazelcastNode.hazelcastInstance.getDistributedObjects();
+        caches = distributedObjects.stream().filter(distributedObject -> distributedObject.getServiceName().
+                equals(CacheService.SERVICE_NAME)).collect(toList());
+        Map<String, String> cacheInfo = new HashMap<>();
```

Comment:
Since there is just 1 item in the map, I think we can set the initial capacity for that map correctly.

Reply:
I think property should be implemented as a diagnostics. Now we get an ad hoc addition to the system to collect data.